observability/monitoring/monitoring-argo-cd-custom-resource-workloads.adoc

```diff
@@ -34,4 +34,4 @@ include::modules/gitops-disabling-monitoring-for-argo-cd-custom-resource-workloa
 [role="_additional-resources"]
 [id="additional-resources_monitoring-argo-cd-custom-resource-workloads"]
 == Additional resources
-* link:https://docs.openshift.com/container-platform/latest/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
+* link:https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
```

Comment:
```suggestion
* link:https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects]
```

Reply:
@Dhruv-Soni11 L37 not fixed properly. Please add the .adoc instead of .html.

Reply:
Fixed it now @Srivaralakshmi. PTAL.

Reply:
@Dhruv-Soni11 I am sorry, my brain is clogged.. it is not an xref, it is a link.. so html is right. Sorry, please scratch what I said earlier about .adoc. Restore your original statement.